tests/datasets/test_transaction_processor.py

```diff
@@ -16,6 +18,7 @@ class TransactionEvent:
     event_id: str
     trace_id: str
     span_id: str
+    group_ids: list[int]
```

Comment:
Instead of a list, you can use Sequence over here

Reply:
done!